OOStepByStep/Class.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOStepByStep
+{
+    public class Class
+    {
+        private int? classNo = null;
+        private Teacher? teacher = null;
+        private List<Student> students = new List<Student>();
+
+        public Class(int classNo, Teacher teacher)
+        {
+            this.classNo = classNo;
+            this.teacher = teacher;
+        }
+
+        public string AddNewStudent(Student student)
+        {
+            student.ClassNo = classNo;
+            string teacherWelcome = this.teacher.SelfIntroduce() + " Welcome " + student.Name + " join class " + student.ClassNo + ".";
+
+            List<string> studentsWelcomeList = new ();
+
+            if (students.Count > 0)
```

Comment:
I wonder if the judgement sentence 'if()>0' can be deleted ? Or otherwise it won't compile ?